rest-jersey/src/test/java/org/hammock/test/jersey/JerseyPropertiesTest.java

```diff
@@ -0,0 +1,45 @@
+package org.hammock.test.jersey;
```

Comment:
Can you please add the standard header, to acknowledge that you're licensing it as Apache v2?

Reply:
Ah, yes, I will! Is Apache v2 file-based like CDDL? Or is it more an Apache Foundation standard to mark every file? I'll get this change up in a couple of hours...

Reply:
This is an Apache Foundation project.  The header can be seen on all other files -> https://github.com/hammock-project/hammock/blob/master/rest-jersey/src/test/java/org/hammock/test/jersey/JerseyCustomTest.java

It means that you're licensing the file under ApacheV2, and by contributing to this repo allowing me to release it.  You still retain copyright claims per the header.

Feel free to add your name to the NOTICE file too, if you'd like.

Reply:
Thanks, I was just curious. I knew you were heavily involved in the Apache Foundation, but I didn't realize Hammock was also an official project!

I guess my eyes and my IDEs are good at collapsing boilerplate!

I'm not concerned with licenses or any of that, except that the GPL will win out in the end as a logical inevitability! ;) j/k

I'm happy to comply with project standards, I'm just not at a computer terminal right now.

Reply:
Oh, I misread! You said *not* an Apache project...

Reply:
I definitely understand the need to acknowledge the expectations of contribution.

Speaking of... I've also noticed you mark some commits with `Signed-off-by`. Are you also committed to maintaining that level of auditability of this body of work?

IIRC, `Signed-off-by` is an important legal instrument for granting permission to use a copyright in a body of work, almost like the opposite of a copyright assignment, it's a copyright allowance or developer certificate of origin.

Reply:
I had a typo first time I responded so I edited it :-D

I tend to use sign off when I remember.  Before my computer died, I was signing via GPG keys.  This is a loaner, hopefully once the other one comes back I'll be good to go again.

Reply:
More on the meaning of `Signed-off-by` here: https://www.youtube.com/watch?v=vyenmLqJQjs

Probably you should require it from all contributors. Probably means you need to include the Developer Certificate of Origin in the repo...